app/src/main/res/xml/preferences.xml

```diff
@@ -26,6 +26,12 @@
     </PreferenceCategory>
 
     <PreferenceCategory android:title="@string/pref_general">
+        <ListPreference
+            android:defaultValue="Theme_Jellyfin"
+            android:entries="@array/app_theme_entries"
+            android:entryValues="@array/app_theme_values"
+            android:key="app_theme"
+            android:title="@string/pref_app_theme" />
```

Comment:
Adding `app:useSimpleSummaryProvider="true"` will show the currently selected theme as "description".